### PR TITLE
Add transition on hover for side bar item

### LINKED ISSUE #1958: `VaSideBarItem` add transition on hover

### PR DESCRIPTION
**Pull request by arkmadj**
## Description
Add transition on hover for sidebar item

<details>

```vue
      // add transition on isHovered for background color
      if (isHovered.value || props.active || hasKeyboardFocus.value) {
        style.backgroundColor = backgroundColorComputed.value
        style.transition = 'background-color 0.6s ease-in-out'
      }
     
      ...

      // add transition for scss too for text color
      transition: 0.6s ease-in-out;
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

closes #1958

### CODE REVIEW

File: packages/ui/src/components/va-sidebar/VaSidebarItem/VaSidebarItem.vue [hunk 1]
```diff
@@ -71,6 +71,7 @@ export default defineComponent({
 
       if (isHovered.value || props.active || hasKeyboardFocus.value) {
         style.backgroundColor = backgroundColorComputed.value
+        style.transition = 'background-color 0.6s ease-in-out'
```
Comment (RVitaly1978) [inline packages/ui/src/components/va-sidebar/VaSidebarItem/VaSidebarItem.vue:74]:
```suggestion
        style.transition = 'background-color var(--va-sidebar-item-transition)'
```

**Review by RVitaly1978 — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/va-sidebar/VaSidebarItem/VaSidebarItem.vue (hunk 1 above)
Comment (arkmadj) [reply packages/ui/src/components/va-sidebar/VaSidebarItem/VaSidebarItem.vue:74]: Okay. I would update my code with these suggestions. Thanks

**Review by rustem-nasyrov — APPROVED** [review]

**Review by aluarius — APPROVED** [review]

File: packages/ui/src/components/va-sidebar/VaSidebarItem/VaSidebarItem.vue (hunk 1 above)
Comment (m0ksem) [reply packages/ui/src/components/va-sidebar/VaSidebarItem/VaSidebarItem.vue:74]: We also have global `--va-transition` variable.

It's even better to create
```
--va-sidebar-item-transition: var(--va-transition);
```

This way, user can change transition for ALL components in one place by changing `--va-transition` or change transition specific for sidebar by changing `--va-sidebar-item-transition`.

Sorry for not writing it in issue.

**Review by m0ksem — CHANGES_REQUESTED** [review]

**Comment by m0ksem** [comment]
I will update your PR with CSS variables and resolve conflicts. Thanks!

**Pull request merged**